README.md

```diff
@@ -123,25 +133,53 @@ The important features are:
 
 ### 4. Run the pipeline 
 
-The most basic run command for this pipeline is: 
+The most basic run command for this pipeline with all required paramters looks like this: 
 
 ```
-nextflow run main.nf --input samples.csv --ref reference.fasta  --intervalList_path path_to_intervals --ponvcf pon.vcf.gz
-        
+nextflow run main.nf --input samples.csv \
+                     --ref /path/to/ref.fasta --dict /path/to/ref.dict \
+                     --ponvcf /path/to/pon \
+                     --common_biallelic_variants /path/to/common_biallelic_variants
 ```
-**Note**: Please use the command provided in the script `scripts/run_pipeline_on_gadi_script.sh` on NCI Gadi HPC.
 
 By default, this will generate `work` directory, `results` output directory and a `runInfo` run metrics directory inside the results directory. 
 
+
+This pipeline has been tested for NCI Gadi HPC and the paramaters `--whoami` and `--gadi_account` will have to be included as discussed in the section [Infrastructure usage and recommendations](#infrastructure-usage-and-recommendations) .
+```
+nextflow run main.nf --input samples.csv \
+                     --ref /path/to/ref.fasta --dict /path/to/ref.dict \
+                     --ponvcf /path/to/pon \
+                     --common_biallelic_variants /path/to/common_biallelic_variants \
+                     -profile gadi \
+                     --whoami $(whoami) --gadi_account $PROJECT    
+```
+
+
 To specify additional optional tool-specific parameters, see what flags are supported by running:
 
 ```
 nextflow run main.nf --help 
 ```
 
+A more advanced nextflow command with optional parameters, where you can define the name of the output folder and provide an integer value for the number of genomic-interval files to be generated looks like this:
+```
+nextflow run main.nf --input samples.csv \
+                     --ref /path/to/ref.fasta --dict /path/to/ref.dict \
+                     --ponvcf /path/to/pon \
+                     --common_biallelic_variants /path/to/common_biallelic_variants \
+                     -profile gadi \
+                     --whoami $(whoami) --gadi_account $PROJECT \
+                     --outDir name_output_folder \
+                     --number_of_intervals INT
+```
+
+
 If for any reason your workflow fails, you are able to resume the workflow from the last successful process with `-resume`. 
 
 
+
+
 ### NCI Gadi HPC
```

Comment:
Currently you have information required to execute workflow on Gadi split across various parts of the README. You need to commit to one part of the README that will contain all of this information. See comment above re: infrastructure usage recommendations section.